webextensions/manifest/options_ui.json

```diff
@@ -62,6 +62,30 @@
               }
             }
           }
+        },
+        "open_in_tab": {
```

Comment:
I don't think this should be a separate subfeature, I think it should just be handled in a note for Opera.

Reply:
Where should that note be? Inside [the Syntax table](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/options_ui#Syntax)?

IMO this Opera incompatibility is very surprising, I thought I could strip out some superfluous styles from my options pages... and then I tested Opera :(

Reply:
I'm sorry, I totally missed that `open_in_tab` is actually a property of `option_ui`. Your PR makes a lot more sense to me now :).

So I think having `open_in_tab` as a subfeature is just fine. I still think the note you've added:

    "Does not support inline extension options, it always opens it in a separate tab."

....should be attached to the `options_ui` compat statement, i.e. here: https://github.com/mdn/browser-compat-data/blob/master/webextensions/manifest/options_ui.json#L20.

The reason being that, as I understand it, Opera changes the *default behavior* of `options_ui`: that is, if I omit `open_in_tab` I will still get different behavior in Opera. So we should note that alongside `options_ui`.

Does that make sense?